source/shared/cpp/ObjectModel/Column.cpp

```diff
@@ -112,4 +112,14 @@ std::shared_ptr<Column> Column::Deserialize(const Json::Value& value)
 std::shared_ptr<Column> Column::DeserializeFromString(const std::string& jsonString)
 {
     return Column::Deserialize(ParseUtil::GetJsonValueFromString(jsonString));
+}
+
+std::shared_ptr<BaseActionElement> Column::GetSelectAction() const
```

Comment:
General comment:  Where is the serialization and deserialization logic for all three classes?